media/css/firefox/new/basic/platform.scss

```diff
@@ -7,6 +7,10 @@ $image-path: '/media/protocol/img';
 
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 
+.c-linux-debian {
+    margin-top: $spacing-md;
+}
+
```

Comment:
Issue: at smaller breakpoints the download buttons on this page are missing some padding

![image](https://github.com/mozilla/bedrock/assets/400117/4064bbdf-9b80-40b5-a5d0-244bb06314a7)